transformers/events/cat_file/vow/transformer_test.go

```diff
@@ -57,10 +61,14 @@ var _ = Describe("Cat file vow transformer", func() {
 		Expect(err).To(HaveOccurred())
 	})
 
-	It("converts a log to an model", func() {
-		models, err := transformer.ToModels(constants.CatABI(), []core.EventLog{test_data.CatFileVowEventLog}, nil)
+	It("converts a log to a model", func() {
+		models, err := transformer.ToModels(constants.CatABI(), []core.EventLog{test_data.CatFileVowEventLog}, db)
 
 		Expect(err).NotTo(HaveOccurred())
-		Expect(models).To(Equal([]event.InsertionModel{test_data.CatFileVowModel()}))
+		expectedModel := test_data.CatFileVowModel()
+		msgSenderID, msgSenderErr := shared.GetOrCreateAddress(test_data.CatFileVowEventLog.Log.Topics[1].Hex(), db)
```

Comment:
You could use test_data.AssignMessageSenderID here - Yes I'm sorry I'm a broken record.